packages/terra-hookshot/src/HookshotContent.jsx

```diff
@@ -77,6 +77,7 @@ class HookshotContent extends React.Component {
 
   handleClickOutside(event) {
     if (this.props.onOutsideClick) {
+      event.preventDefault();
```

Comment:
Are there cases where we wouldn't want to preventDefault? If so we might want to look at adding a prop to control this. I can't think of any cases at this time.

Reply:
So if i click outside of an open popup it will prevent default behavior on that click? I actually can't think of a situation when that is desirable?  Example:

* Clicking and focusing on a form input
* Clicking a link
* Clicking on another thing to trigger a popup

In all these cases I'd want hookshot to disappear and the action i triggered to just happen. IMO if we need to prevent outside clicks, that should be done by a overlay that's paired with this.